src/units/unit.hpp

```diff
@@ -1774,6 +1774,8 @@ class unit
 	bool emit_zoc_;
 
 	std::vector<std::string> overlays_;
+	std::vector<std::string> meta_overlays_;
+	std::vector<std::string> obj_overlays_;
```

Comment:
Please add some documentation about what the difference between these three vectors is. I know Wesnoth is light on documentation, but it helps a code review to know what the three things should be.